boardgame-ui/src/pages/EventCreationPage.vue

```diff
@@ -42,9 +42,7 @@
         </b-field>
 
         <b-field horizontal>
-          <button type="submit" class="button is-primary">
-            {{$t('event.edition.submit')}}
-          </button>
+          <button type="submit" class="button is-primary">{{$t('event.edition.submit')}}</button>
```

Comment:
you should use the `<i18n>` tag to inject localized content (see in home page for an exemple).

Reply:
According to @elodieburtin you can use the $t('...) syntax just fine. <i18n> is only really needed when you need to inject some html into the translation.